types/foundry-vtt-api/types/framework/entities/macro.d.ts

```diff
@@ -0,0 +1,45 @@
+/**
+ * The Collection of Macro entities
+ * @extends {Collection}
+ */
+declare class Macros extends Collection<Macro> {
+	entities: Macro[];
```

Comment:
<!--
  1 failure:  The root .editorc...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: types/foundry-vtt-api/types/framework/entities/macro.d.ts;
  Line: 6;
-->
- :no_entry_sign: The root .editorconfig style specifies spaces for whitespace. Please use spaces in new or changed types.